### PR TITLE
Fixing automation test

### PR DESCRIPTION
**Pull request by palakpatel311**
Fixing tests for OCD-2916

### CODE REVIEW

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java [hunk 1]
```diff
@@ -119,8 +119,11 @@ public void verifyThatSurveillanceDataIsSavedInReport(final String friendlySurvI
         /* Workaround to resolve Stale Element Reference Exception-
          * reload the element after form update so it's attached to DOM: https://www.seleniumhq.org/exceptions/stale_element_reference.jsp*/
         getDriver().get(getUrl() + "#/surveillance/reporting");
+        getDriver().navigate().refresh();
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java:122]: Why does the page need a refresh immediately after load? I see the comment about the workaround; does this actually work? I don't know why this page, specifically, would be any different on initial load than after a refresh

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java [hunk 2]
```diff
@@ -119,8 +119,11 @@ public void verifyThatSurveillanceDataIsSavedInReport(final String friendlySurvI
...
         steps.iExpandACB("UL LLC");
-        SurveillanceReportPage.editSurveillanceReport(getDriver(), "UL LLC-2019-Q1").click();
+        WebElement editbutton = SurveillanceReportPage.editSurveillanceReport(getDriver(), "UL LLC-2019-Q1");
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java]: This quarter isn't an editable quarter. That's fine to use it, but the PageElement indicates it's editable, not initiating, which is confusing

Has it been initiated in a previous step?

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java [hunk 3]
```diff
@@ -119,8 +119,11 @@ public void verifyThatSurveillanceDataIsSavedInReport(final String friendlySurvI
...
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].scrollIntoView(true);", editbutton);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", editbutton);
```
Comment (andlar) [inline src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java]: After clicking this button, there's a popup about "are you sure you want to initiate surveillance", so the next few lines can't happen

File: src/test/resources/Features/UI/OCD-2806-SimplifiedEditingOfVersion.feature [hunk 4]
```diff
@@ -4,8 +4,8 @@ Feature: OCD-2806 Simplified Editing of Version
 
 Scenario Outline: User logs in as ROLE and sees edit version link exists on new developer page
     Given I'm logged in as "<ROLE>"
-    When I navigate to the version edit page for version "478" of product "1742" of developer "1816"
-    Then I see edit link for version "478"
+    When I expand product "Agility EHR" of developer "1816" to view version dropdown
```
Comment (andlar) [inline src/test/resources/Features/UI/OCD-2806-SimplifiedEditingOfVersion.feature]: Are there new step definitions for these steps?

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java (hunk 1 above)
Comment (palakpatel311) [reply src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java:122]: It was throwing stale element so i had to refresh page after i initiate report.

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java (hunk 2 above)
Comment (palakpatel311) [reply src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java]: Yes. It has been initiated in And I confirm initiating a quarterly surveillance report step.

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java (hunk 3 above)
Comment (palakpatel311) [reply src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java]: This is for edit button, not initiate

Comment (palakpatel311) [reply src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java]: It is for edit button and not initiate

File: src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java (hunk 1 above)
Comment (andlar) [reply src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java:122]: But this isn't refreshing after initiating the report; it's refreshing after loading the page

Comment (palakpatel311) [reply src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java:122]: Cleaned the code

**Review by andlar — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Pull request merged**